client/src/banners/active-banner.tsx

```diff
@@ -64,47 +61,21 @@ function Banner(props: BannerProps) {
   );
 }
 
-// function CommonSurveyBanner({ onDismissed }: { onDismissed: () => void }) {
-//   const ga = useGA();
-
-//   return (
-//     <Banner
-//       id={COMMON_SURVEY_ID}
-//       title={"Learning web development survey"}
-//       copy={
-//         "Help us understand how to make MDN better for beginners (5 minute survey)"
-//       }
-//       cta={"Take the survey"}
-//       url="https://www.surveygizmo.com/s3/6175365/59cfad9c04cf"
-//       newWindow
-//       onDismissed={onDismissed}
-//       onCTAClick={() => {
-//         ga("send", {
-//           hitType: "event",
-//           eventCategory: CATEGORY_LEARNING_SURVEY,
-//           eventAction: "CTA clicked",
-//           eventLabel: "banner",
-//         });
-//       }}
-//     />
-//   );
-// }
+function SendToGA(eventCategory: string) {
```

Comment:
Since this function doesn't return a component, it should probably be lowercase, and I would make it more descriptive:

```suggestion
function sendCTAEventToGA(eventCategory: string) {
```